assets/stylesheets/units/_buttons.scss

```diff
@@ -0,0 +1,43 @@
+.button {
+  @include bold-font(20);
+
+  -webkit-appearance: none;
+  background-color: shade($green, 12.5%);
+  border: 0;
+  color: $white;
+  cursor: pointer;
+  display: inline-block;
+  letter-spacing: .5px;
+  margin: 0 ($baseline-grid-unit * 6) ($baseline-grid-unit * 3) 0;
```

Comment:
Instead of always including a bottom margin we should use `* + .button` with a top margin which only sets a margin if it is proceeded by another element.

Also consider using the `$default-spacing-unit` for that spacing and maybe only on mobile?